internal/persistence/sql/migrations/templates/20220110200400_create-uuid-mapping-table.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE keto_uuid_mappings
+(
+    id                       UUID NOT NULL,
+    string_representation    TEXT NOT NULL CHECK (string_representation <> ''),
+
+    PRIMARY KEY (id)
+);
```

Comment:
I just realized that we of course have to change the column types in the relation tuple table to UUID as well. I am afraid that we have to create a new table and insert into that instead of the old one. Or do you have other ideas?
The migrator we use can theoretically be extended to add go migrations btw, I think that would definitely make sense. Then we also don't need the command :thinking:
What do you think about that @aeneasr?

https://github.com/ory/x/blob/fdc336251a395d29c7a242b611dc3fe3740620f8/popx/migration_box.go

Reply:
Creating a new table might not be necessary. Assuming that `object` and `subject_id` already hold UUIDs, we can, in Postgres, do:
```sql
ALTER TABLE keto_relation_tuples ALTER object TYPE UUID using object::uuid;
```
(and similar for `subject_id`). MySQL does not have a UUID type, so we will stick with `VARCHAR(64)`. The same statement works in Cockroachdb when using `SET enable_experimental_alter_column_type_general = true`.